webpack.config.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
     new CopyPlugin([
       { from: 'manifest.konnector' },
       { from: 'package.json' },
+      { from: 'yarn.lock' },
```

Comment:
With the commit message, I thought you added `yarn build` to the the yarn script "build". OK for me though.